File: contest/LeetCode_Weekly_Contest_202/Three_Consecutive_Odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int n = arr.size();
        for (int i = 0; i + 3 <= n; ++i) {
            if (arr[i]&1 && arr[i+1]&1 && arr[i+2]&1) 
                return true;
        }
        return false;
    }
};

File: contest/LeetCode_Weekly_Contest_202/Three_Consecutive_Odds.js
/**
 * @param {number[]} arr
 * @return {boolean}
 */
var threeConsecutiveOdds = function(arr) {
    var n = arr.length;
    for (var i = 0; i+3 <= n; i++) {
        var flag = true;
        for (var k = 0; k < 3; k++) {
            if (arr[i+k] % 2 != 1) {
                flag = false;
                break;
            }
        }
        if (flag) return true;
    }
    return false;
};
